YotiButtonSDK/Kernel/KernelSDK.swift

```diff
@@ -21,13 +21,17 @@ class KernelSDK: NSObject {
      * Perform a call to the Yoti API to retrieve the scenario and start the Yoti App
      */
     func startScenario(for useCaseID: String) {
-        
+
+        NotificationCenter.default.post(name: YotiSDK.willMakeNetorkRequest, object: nil)
```

Comment:
Better include `useCaseID` or the `scenario` object in the notification as a reference between different requests.